subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/PmdExtension.java

```diff
@@ -101,6 +102,26 @@ public void setTargetJdk(TargetJdk targetJdk) {
         this.targetJdk = targetJdk;
     }
 
+    /**
+     * The maximum number of failures to allow before stopping the build.
+     *
+     * If <pre>ignoreFailures</pre> is set, this is ignored and no limit is enforced.
+     */
+    public int getMaxFailures() {
+        return maxFailures;
+    }
+
+    /**
+     * The maximum number of failures to allow before stopping the build.
+     *
+     * If <pre>ignoreFailures</pre> is set, this is ignored and no limit is enforced.
+     *
+     * @param maxFailures The maximum number of failures to allow.
+     */
```

Comment:
These need `@since` tags and `@Incubating` annotations to mark them as new APIs (thus the compatibility failures in our last build)

Reply:
added.  I put 6.3 on the since because that's the label on this issue, hope that's right.